cypress-io/cypress
# refactor: use cp.fork to preserve stdout for packages/extension build

### User facing changelog
na

### Additional details
The extension package executes webpack inside of a gulp script. It was using a promisified `exec` script to invoke the `run-webpack.js` script and it wasn't piping any of the stdout. This PR uses `fork` to invoke webpack, making it easy to capture stdout with `stdio: 'inherit'`.

I ran into this when working through adding Angular CT support. Webpack 5 got hoisted and broke the webpack build but the error message was unclear. Now, if this script errors it will print the webpack logs.

### Steps to test
Run `yarn workspace @packages/extension build` and you'll see the webpack logs. You can tweak the src files to throw a compilation error to verify that a non-zero exit code causes the gulp task to fail.

### How has the user experience changed?
Before:
![Screen Shot 2022-07-25 at 5 53 53 PM](https://user-images.githubusercontent.com/25158820/180887553-d81b70be-a6b7-468d-a6cf-954f4bb00d20.png)


After:
![Screen Shot 2022-07-25 at 5 47 52 PM](https://user-images.githubusercontent.com/25158820/180886944-d665db87-97e7-4b07-a3bf-aaae77fa5e01.png)


### PR Tasks


- [na] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
